//Test Script

/* 
integer age = 25;
system.debug(age);
}
*/

PLACEHOLDER

public class customException extends exception{}
try{
system.assertEquals(25, age);
}
catch(exception exc){
throw new customException('there is an error!' +exc.getMessage()); 
}